rust/basic_bitcoin/basic_bitcoin.did

```diff
@@ -57,6 +59,10 @@ service : (network) -> {
 
   "get_current_fee_percentiles" : () -> (vec millisatoshi_per_vbyte);
 
+  "inscribe_brc20": (ticker: text) -> (transaction_id);
```

Comment:
Is BRC-20 actually still relevant enough to have a dedicated example?

Reply:
Apparently it is. I have heard devs talking about a resurgence.